### PR TITLE
Update app.vue

### PR DESCRIPTION
**Pull request by amiiiiii830**

### CODE REVIEW

**Comment by amiiiiii830** [comment]
more changes on the way

**Comment by jaykchen** [comment]
ok

**Comment by amiiiiii830** [comment]
yarn.lock

**Comment by jaykchen** [comment]
ok great

File: yarn.lock [hunk 1]
```diff
@@ -1,4 +1,3 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (jaykchen) [inline yarn.lock:1]: you can change it as you want

**Review by jaykchen — APPROVED** [review]
pass

**Comment by amiiiiii830** [comment]
thanks

**Pull request closed without merging**